contextualSpellCheck/contextualSpellCheck.py

```diff
@@ -265,6 +163,98 @@ def misspell_identify(self, doc, query=""):
             print("misspell identified: ", misspell)
         return misspell, doc
 
+    def __set_spacy_extensions(self):
+        if not Doc.has_extension("contextual_spellCheck"):
+            Doc.set_extension("contextual_spellCheck", default=True)
+            Doc.set_extension("performed_spellCheck", default=False)
+
+            Doc.set_extension("suggestions_spellCheck", default={})
+            Doc.set_extension("outcome_spellCheck", default="")
+            Doc.set_extension("score_spellCheck", default=None)
+
+            Span.set_extension(
+                "get_has_spellCheck", getter=self.span_require_spell_check
+            )
+
+            Span.set_extension(
+                "has_spelling_correction", getter=self.span_require_spell_check
+            )
+
+            #
+            Span.set_extension(
+                "score_spellCheck", getter=self.span_score_spell_check
+            )
+
+            # Warning: Deprecated, use `has_spelling_correction`
+            Token.set_extension(
+                "get_require_spellCheck", getter=self.token_require_spell_check
+            )
+
+            Token.set_extension(
+                "has_spelling_correction", getter=self.token_require_spell_check
+            )
+
+            Token.set_extension(
+                "get_suggestion_spellCheck",
+                getter=self.token_suggestion_spell_check,
+            )
+            Token.set_extension(
+                "score_spellCheck", getter=self.token_score_spell_check
+            )
+
+    def __get_vocab_words(self):
+        vocab_path = self.user_provided_vocab_path
+        words = []
+
+        if vocab_path != "":
+
+            try:
+                # First open() for user specified word addition to vocab
+                with open(vocab_path, encoding="utf8") as vocab_file_buffer:
```

Comment:
You are using `utf8` too many time. maybe its better to make it as const param like this.
`FORMAT_UTF8 = 'utf8'`